src/Polling/Frequent/FrequentPollingActivity.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.Extensions.Logging;
+using Temporalio.Activities;
+using Temporalio.Exceptions;
+using TemporalioSamples.Polling.Common;
+
+namespace TemporalioSamples.Polling.Frequent;
+
+public class FrequentPollingActivity : IPollingActivity
```

Comment:
```suggestion
public class FrequentPollingActivities
```

I don't have a strong opinion here, but I have been avoiding assuming a class is an activity since it can have many (activities are methods)